packages/wallet/frontend/src/components/layouts/AuthLayout.tsx

```diff
@@ -26,24 +26,22 @@ const AuthLayout = ({ image, children }: AuthLayoutProps) => {
   const imageSrc = `/${IMAGES[imageName as Image]}`
   return (
     <>
-      <div className="h-full">
-        <div className="flex min-h-full">
-          <div className="relative hidden w-0 flex-1 md:block">
-            <Image
-              fill
-              // sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
-              className="object-cover"
-              src={imageSrc}
-              alt={image}
-              quality={100}
-              priority={true}
-              loading="eager"
-            />
-          </div>
-          <div className="min-h-full flex-1 py-10">
-            <div className="mx-auto flex min-h-full w-full flex-col items-center sm:px-6 lg:px-20 xl:px-24">
-              {children}
-            </div>
+      <div className="flex min-h-full w-screen">
+        <div className="relative hidden w-0 flex-1 md:block">
+          <Image
+            fill
+            // sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
+            className="object-cover"
+            src={imageSrc}
```

Comment:
<img width="432" alt="image" src="https://github.com/user-attachments/assets/3ed2b5f3-11f0-418a-9782-2b7daba10233">

This image does not seem right also. I'm not seeing this in the Figma file.

Reply:
My birdie looks like this:
<img width="354" alt="image" src="https://github.com/user-attachments/assets/b9cefb40-2101-44c8-ba13-7e34d310df82">